pom-redhat.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2016-2017 Red Hat, Inc, and individual contributors.
+  ~
+  ~  Licensed under the Apache License, Version 2.0 (the "License");
+  ~  you may not use this file except in compliance with the License.
+  ~  You may obtain a copy of the License at
+  ~
+  ~  http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~  Unless required by applicable law or agreed to in writing, software
+  ~  distributed under the License is distributed on an "AS IS" BASIS,
+  ~  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~  See the License for the specific language governing permissions and
+  ~  limitations under the License.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.openshift</groupId>
+    <artifactId>booster-parent</artifactId>
+    <version>10</version>
+  </parent>
+
+  <groupId>io.openshift.booster</groupId>
+  <artifactId>wfswarm-rest-http-secured-parent</artifactId>
+  <version>5-SNAPSHOT</version>
```

Comment:
Maybe wanna change this to `7.0.0-redhat-1-SNAPSHOT`?

Reply:
good point

Reply:
Actually I think this will settle when `pom-redhat.xml` turns into `pom.xml`

Reply:
This is the same issue as with the CB booster.

Reply:
It's not the same issue as with the CB booster, but it solves the same way: once you first release (which includes setting the correct version), it will be fine.